this_the_way.py

```diff
@@ -0,0 +1,11 @@
+import os
+
+
+def rename_files(path):
```

Comment:
1. Please use a better name for the function - are we renaming files in it?
2. There is missing documantation, please add it.
3. Consider changing the name of "path" variable to a better one.